modules/client/src/addresslistVoting/internal/client/estimation.ts

```diff
@@ -30,11 +25,6 @@ export class AddresslistVotingClientEstimation extends ClientCore
     params: CreateMajorityVotingProposalParams,
   ): Promise<GasFeeEstimation> {
     const signer = this.web3.getConnectedSigner();
-    if (!signer) {
-      throw new NoSignerError();
-    } else if (!signer.provider) {
```

Comment:
Didn't we just add some lines with `getProvider()` on the other PR? Aren't we going to miss these checks if so?

Reply:
Mmm no I dont think so, this checks no longer makes sense becuse they are already dont by the `getConnectedSigner` function that calls `getProvider` and `getSigner` and both these functions check if the provider and signer exist